### PR TITLE
Added python script for server sign operations 

### PR DESCRIPTION
**Pull request by ShubhadaHarkut-eaton**
Removed the Bat file support and added the server_sign_image.py for server sign operations.

### CODE REVIEW

File: scripts/imgtool/Server_Sign/server_sign_image.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        sys.exit()
+    else:
+        print("Input File: " + str(file_path) + "\n")
+        username = input("Enter User Name (Eaton ID):")
+        if username.strip() =='':
+            print("NO User Entered \n ")
+            print(" Process Terminated ")
+            sys.exit()
+        else:
+            server_home = os.environ.get('SERVER_HOME')
```
Comment (prayassamriya) [inline scripts/imgtool/Server_Sign/server_sign_image.py:18]: SIGN_SERVER_HOME
SIGN_SERVER_WORKER

File: scripts/imgtool/Server_Sign/server_sign_image.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            worker = os.environ.get('SERVER_WORKER')
+            if server_home == 'None' or worker == None :
+                print("Please set the enviornment variables SERVER_HOME and SERVER_WORKER ")
```
Comment (prayassamriya) [inline scripts/imgtool/Server_Sign/server_sign_image.py:21]:
```suggestion
                print("Please set the enviornment variables SIGN_SERVER_HOME and SIGN_SERVER_WORKER ")

```

**Review by prayassamriya — APPROVED** [review]

File: scripts/imgtool/Server_Sign/server_sign_image.py (hunk 1 above)
Comment (ShubhadaHarkut-eaton) [reply scripts/imgtool/Server_Sign/server_sign_image.py]: fixed

File: scripts/imgtool/Server_Sign/server_sign_image.py (hunk 2 above)
Comment (ShubhadaHarkut-eaton) [reply scripts/imgtool/Server_Sign/server_sign_image.py]: fixed

File: scripts/imgtool/Server_Sign/server_sign_image.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        if username.strip() =='':
+            print("NO User Entered \n ")
+            print(" Process Terminated ")
+            sys.exit()
+        else:
+            server_home = os.environ.get('SIGNSERVER_HOME')
+            worker = os.environ.get('SIGNSERVER_WORKER')
+            if server_home == None or worker == None :
+                print("Below enviornment variables are not found")
+                print("SIGNSERVER_HOME  - should be the path of sign server utility installed on your system \n ")
```
Comment (prayassamriya) [inline scripts/imgtool/Server_Sign/server_sign_image.py:24]:
```suggestion
                print("\nSIGNSERVER_HOME  - should be the path of sign server utility installed on your system \n ")

```

File: scripts/imgtool/Server_Sign/server_sign_image.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    # Strips the newline character
+                    for line in Lines:
+                        count += 1
+                        if 'OCTET STRING' in line:
+                            address = line.partition(':')[0]
+                    print("address : " + str(address))
+                    sig_extract_cmd = "openssl asn1parse -inform DER -in " + str(output_file) + " -out " + str(output_file) + ".sig" + " -noout -strparse " + str(address)
+                    subprocess.run(sig_extract_cmd)
+                else :
+                    print("Please check the path for SIGN_SERVER_HOME bin folder not found")
```
Comment (prayassamriya) [inline scripts/imgtool/Server_Sign/server_sign_image.py:54]:
```suggestion
                    print("Please check the path for SIGNSERVER_HOME bin folder not found")

```

**Review by prayassamriya — APPROVED** [review]

**Pull request merged**